src/pages/resultsView/mutation/Mutations.tsx

```diff
@@ -238,6 +242,33 @@ export default class Mutations extends React.Component<
         }
     }
 
+    @autobind
+    @action
```

Comment:
this doesn't need to be @action because it doesn't do update mobxstate.  it just returns data, so it's actually not an "action".